Source/WebCore/platform/audio/cocoa/MediaSessionManagerCocoa.mm

```diff
@@ -336,6 +336,24 @@
             updateNowPlayingInfo();
         });
     }
+
+#if USE(AUDIO_SESSION)
+    // De-activate the audio session if the last playing session is:
+    // * An audio presentation
+    // * Is in the ended state
+    // * Has a short duration
+    // This allows other applications to resume playback after an "alert-like" audio
+    // is played by web content.
+
+    // Derived from the duration of the "You've got mail!" AOL sound:
+    static constexpr MediaTime YouveGotMailDuration = MediaTime(95, 100);
```

Comment:
If you put `isLongEnoughForMainContent` in `PlatformMediaSession ` you could use it from `MediaElementSession` and not define the constant in more than one place